user-profile-tag-list-item.js

```diff
@@ -0,0 +1,101 @@
+import 'd2l-users/components/d2l-profile-image.js';
+import './user-profile-card.js';
+import { css, html, LitElement } from 'lit-element';
+import { LocalizeUserProfileCard } from './localize-user-profile-card.js';
+import { RtlMixin } from '@brightspace-ui/core/mixins/rtl-mixin.js';
+import { TagListItemMixin } from '@brightspace-ui/core/components/tag-list/tag-list-item-mixin.js';
+import { UserDisplayNameMixin } from 'd2l-users/components/d2l-user-display-name-mixin.js';
+
+class UserProfileTagListItem extends RtlMixin(UserDisplayNameMixin(LocalizeUserProfileCard(TagListItemMixin(LitElement)))) {
+
+	static get styles() {
+		return [super.styles, css`
+			.profile-image {
+				display: inline-block;
+                height: 1.2rem;
+				left: 0.15rem;
+				position: absolute;
+				top: 50%;
+				transform: translate(0, -50%);
+                width: 1.2rem;
+			}
+			:host([dir="rtl"]) .profile-image {
+				left: unset;
+				right: 0.15rem;
+			}
+			.display-name {
+				padding-left: calc(1.55rem - 0.6rem);
+				vertical-align: middle;
+			}
+			:host([dir="rtl"]) .display-name {
+				padding-left: 0;
+				padding-right: calc(1.55rem - 0.6rem);
```

Comment:
```suggestion
				padding-left: calc(1.7rem - 0.6rem);
				vertical-align: middle;
			}
			:host([dir="rtl"]) .display-name {
				padding-left: 0;
				padding-right: calc(1.7rem - 0.6rem);
```
I think the spec might have changed since I originally did these calculations (or I just did them wrong originally lol).  But looks like we want  the name to be `3px` + `24px` + `7px` = `1.7rem` from the left edge (and we minus the `0.6rem` padding we get "for free" from the list item mixin).